### ISSUE TITLE
Standard Liter and Norm Liter definitions seem inconsistent with commonly used definitions

### ISSUE THREAD

**Issue by brennmat** [Sensirion/python-shdlc-sfc5xxx#2]
In the documentation, the units for gas amounts are defined as follows:

- The STANDARD_LITER  is defined as "liter of gas at 20°C and 1013hPa.
- The NORM_LITER is defined as "liter of gas at 0°C and 1013hPa.

These definitions seem inconsistent with the commonly used STP and NTP definitions (see for example Wikipedia):
- STP (IUPAC before 1982): T = 0°C and P = 1.0 atm = 1013.25 hPa
- STP (IUPAC after 1982): T = 0°C and P = 1000.0 hPa
- NTP:  T = 20°C and P = 1000.0 hPa (I am not sure about this, though)

In other words, the STANDARD_LITER definition seems to reflect **N**TP, whereas NORM_LITER seems to reflect the pre-1982 **S**TP.

Is this (i) a mixup in the documentation, or (ii) is this simply due to inconsistencies in the "general" definitions of STP and NTP? In the case (ii), it would be useful to add a reference to the documentation linking to the STP/NTP definitions used.

**Comment by ubruhin** [comment]
Hi,

Our naming does not correspond to the STP and NTP definitions as listed on Wikipedia. Instead, the norm liter is defined according DIN 1343 (0°C, 1013.25 hPa). And standard liters does not reflect a clear standard since this term is used differently for different sectors (e.g. gas meters, Semicon, medical use different standard temperatures). In our case it is 20°C as specified in the SFC5xxx datasheets:

![image](https://user-images.githubusercontent.com/5374821/178525506-c63b0142-36bb-4676-b421-a9d835ea4703.png)

...and the documentation in this driver simply represents what the datasheet specifies. Therefore closing this issue. If you still have questions, just re-open the issue.

**Comment by brennmat** [comment]
"the norm liter is defined according DIN 1343 (0°C, 1013.25 hPa). And standard liters does not reflect a clear standard since this term is used differently for different sectors (e.g. gas meters, Semicon, medical use different standard temperatures)."
--> adding this bit of information would be nice so readers of your documentation don't need to get lost in the different conventions/definitions out there.
